src/js/comments.js

```diff
@@ -6,9 +6,22 @@ class Comments {
 	 */
 	constructor (oCommentsEl, opts) {
 		this.oCommentsEl = oCommentsEl;
-		this.options = Object.assign({}, {
+		this.options = Object.assign({}, {}, opts || Comments.getDataAttributes(oCommentsEl));
+		this._renderComments();
+	}
+
+	/**
+	 * Render comment instances and store them on the `_comments` property.
+	 *
+	 * @access private
+	 * @returns {void}
+	 */
+	_renderComments() {
+		const scriptElement = document.createElement('script');
+		scriptElement.src = 'https://ft-next-talk-spike.herokuapp.com/static/embed.js';
+		scriptElement.setAttribute('onload', `Coral.Talk.render(document.querySelector('[data-o-component="o-comments"]'), {talk: 'https://ft-next-talk-spike.herokuapp.com'})`);
```

Comment:
Aren't we supposed to pass a JWT token to the component at this stage?

Reply:
Oh, I'm not sure! @GlynnPhillips, any thoughts?

Reply:
In real life Glynn said: yes, we it does need to be passed in eventually but that can be done at a later time. We are still able to view the comments box - just can't post a comment without the token.